src/microscope_calibration/util/stem_overfocus_sim.py

```diff
@@ -70,8 +70,8 @@ def detector_px_to_specimen_px(
         overfocus, transformation_matrix, fov_size_y, fov_size_x):
     position_y, position_x = (y_px - cy) * detector_pixel_size, (x_px - cx) * detector_pixel_size
     position_y, position_x = transformation_matrix @ np.array((position_y, position_x))
-    specimen_position_y = position_y / camera_length * overfocus
-    specimen_position_x = position_x / camera_length * overfocus
+    specimen_position_y = position_y / (overfocus + camera_length) * overfocus
+    specimen_position_x = position_x / (overfocus + camera_length) * overfocus
```

Comment:
Maybe we can have a docstring that specifically refers to the figure 2 of the paper (or, in the bonus variant, includes it as ASCII art) 

In general a docstring for the parameters would be very helpful (even if marked as private)